example.go

```diff
@@ -0,0 +1,23 @@
+package booleans
+
+
+//canExecuteFastAttack indicates if the knight is awake.
+func canExecuteFastAttack(knightIsAwake bool) bool {
+    return false
```

Comment:
The convention used for stub files is to raise an error:

```suggestion
    panic("Please implement the canExecuteFastAttack function")
```

See the [Go strings exercise](https://github.com/exercism/v3/blob/master/languages/go/exercises/concept/strings/strings.go) for an example.

This comments also applies to the other functions in this file.

Reply:
Correct, just this is the example file and should contain the implementation of the functions, right? :)